lang/java/reef-tang/tang/src/main/java/org/apache/reef/tang/formats/ConfigurationModule.java

```diff
@@ -143,8 +143,24 @@ public final ConfigurationModule set(final Param<? extends Number> opt, final Nu
     return c;
   }
 
+  public final <T> ConfigurationModule setMultipleValues(final Param<T> opt, final Iterable<String> values) {
+    ConfigurationModule c = deepCopy();
```

Comment:
From the code it looks like `deepCopy()` is used so that returned instance of `ConfigurationModule` is a copy of the one on which the method was called, + newly set params. Without it `setMultiple` will forget about any state `ConfigurationModule` had before `setMultiple` call. So calls like `.set(param1).setMultiple(param2)` will actually set only `param2` and forget `param1`.